data/label_descriptions.yaml

```diff
@@ -0,0 +1,84 @@
+    label, which is applied to the virtual machine instance (VMI).
+  label_name: kubernetes_vmi_label_app_kubernetes_io_name
+- label_description: The name of the Kubernetes node to which the virtual machine
+    instance (VMI) is assigned.
+  label_name: kubernetes_vmi_label_kubevirt_io_nodeName
+- label_description: The name or identifier of the disk being monitored.
+  label_name: disk_name
+- label_description: The type of filesystem used by the disk or partition being monitored
+    (e.g., ext4, xfs, ntfs).
+  label_name: file_system_type
+- label_description: The directory path where the disk or filesystem is mounted.
+  label_name: mount_point
+- label_description: The name of the network interface on the virtual machine or physical
+    machine being monitored.
+  label_name: interface
+- label_description: Indicates the direction of network traffic measured by the metric,
+    specifying whether the bytes are being received or transmitted by the virtual
+    machine instance (VMI).
+  label_name: type
+- label_description: The operating system running on the monitored virtual machine
+    or physical machine.
+  label_name: os
+- label_description: The predefined type or class of the virtual machine instance,
+    indicating the allocated resources such as CPU, memory, and storage.
+  label_name: instance_type
+- label_description: The version identifier of the guest operating system running
+    inside a virtual machine.
+  label_name: guest_os_version_id
+- label_description: The name of the guest operating system running inside a virtual
+    machine.
+  label_name: guest_os_name
+- label_description: The specific configuration or policy preference applied to a
+    Virtual Machine Instance (VMI) or virtual machine.
+  label_name: preference
+- label_description: The type or architecture of the virtual machine's guest OS machine,
+    such as x86_64 or arm.
+  label_name: guest_os_machine
+- label_description: The current lifecycle phase of a Virtual Machine Instance (VMI),
+    indicating its status within the Kubernetes cluster.
+  label_name: phase
+- label_description: The release version of the kernel running on the guest operating
+    system inside a virtual machine.
+  label_name: guest_os_kernel_release
+- label_description: Indicates the type of workload that the Virtual Machine Instance
+    (VMI) is configured to run, such as a server or a desktop environment.
+  label_name: workload
+- label_description: Indicates the size and resource allocation of the Virtual Machine
+    Instance (VMI), such as the amount of CPU, memory, and sometimes other resources
+    like storage.
+  label_name: flavor
+- label_description: The name or identifier of a storage drive or disk attached to
+    the virtual machine or physical machine.
+  label_name: drive
+- label_description: The identifier or index of the CPU core being monitored, typically
+    within a virtual machine or physical machine.
+  label_name: id
```

Comment:
Does setting the description here override it for all series with this label?

Also, I wonder if per-label metadata is actually needed? This isn't there in prometheus so we would be adding this manually anyway, and most of these are understandable from their label values.

Reply:
Yes, updating the descriptions here will override it for all series with this label.